### ISSUE TITLE
Unnecessary import in dependencies

### ISSUE THREAD

**Issue by alspirichev** [grab/cocoapods-pod-merge#10]
Hello! Thank you for your work!

I have an issue with dependencies.

This is my `MergeFile`:

```ruby
group 'Networking'
	has_dependencies!
	swift_version! '5.0'

	pod 'Alamofire'
	pod 'AlamofireImage'
	pod 'AlamofireActivityLogger'
end
```

After the merge of these pods I get an Xcode error in `AlamofireImage` files - **No such module 'Alamofire'**.  But if I remove this import statements - everything will be great. (But I have to do it after every `bundle exec pod install`)

**Comment by biocross** [comment]
Ah! This is because the plugin only replaces `<xx/xx>` style imports at this time. I've been meaning to add support for `import x` style imports as well! Will get this in soon! Thank you for reporting.

**Comment by biocross** [comment]
@alspirichev I've implemented this in [Release 0.0.4](https://github.com/grab/cocoapods-pod-merge/releases/tag/0.0.4) 🎉

**Comment by alspirichev** [comment]
oh great!
I think we can close this issue
